build/yamls/antrea-windows-with-ovs.yml

```diff
@@ -17,6 +17,27 @@ data:
       }
       Write-Host "OVSExt driver has been installed"
     }
+
+    # Check if the VC redistributable is already installed.
+    $OVSRedistDir="$mountPath\openvswitch\redist"
+    if (Test-Path $OVSRedistDir) {
```

Comment:
This is because when we are using Windows nanoserver as the base image, we don't deliver the vc_runtime.exe in the images. Instead, we install it in the middle stage, and copy the extracted vcruntime.dll files into nanoserver path "C:\windows\system32\", finally we add the container image's system path into environment Path. So when running containers based on the image, the dll (vcruntime.dll) are accessible.

But for hpc based image, there is no system path like "c:\system\windows" exist in the container image, and we can't extract the dll files during the building stages. So have to copy the exe file into the container image, and install it in init-container stage to extract the dll files on the target Windows Node (host).

If the dll files doesn't exist on the target Windows Node, Windows OVS binary may fail to run.

Reply:
Adding a comment for the `if (Test-Path $OVSRedistDir)` check would be good, but you can do it in your next PR since you want to improve this logic anyway (as discussed in the other comment).